core/modules/file/tests/file_entity.test

```diff
@@ -915,8 +915,8 @@ class FileAltTitleTestCase extends FileTestHelper {
    */
   function testFileAltTitle() {
     $test_file = $this->getTestFile('image');
-    $alt_field_name = 'field_file_image_alt_text'; // Name of the default alt text field added to the image file type.
-    $title_field_name = 'field_file_image_title_text'; // Name of the default title text field added to the image file type.
+    $alt_field_name = 'field_fiat'; // Name of the default alt text field added to the image file type.
```

Comment:
I believe that coding standards mandate that comments are to be placed before the line they are describing.